Todo List Project/assets/js/todoList.js

```diff
@@ -10,24 +9,24 @@ var plusSign = document.getElementById("plusSign");
 
 // addEventListener added to todo list items
 for(var i = 0; i < listItems.length; i++) {
-
-	// deleteBtn();
 	strikeThrough();
 	deleteLine();
 }
 
 addLine();
 showHide();
+plusHide();
 
 // plus sign hides addLine
-
-plusSign.addEventListener("click", function() {
-	if (textIn.style.display === "") {
-		textIn.style.display = "none";
-	} else if (textIn.style.display === "none") {
-		textIn.style.display = "";
-	}
-});
+function plusHide() {
+	plusSign.addEventListener("click", function() {
+		if (textIn.style.display === "") {
```

Comment:
Do you understand the difference between `===` and `==`

Reply:
=== requires an exact match. == requires an approximate match. ex: 0 == "" == null

Reply:
i just assume === is more useful for math and not text values

Reply:
`===` Requires a type match. https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a